packages/frontend/src/components/create-campaign/form/kpi-step/simulation-chart/axis-ticks/reward.tsx

```diff
@@ -0,0 +1,19 @@
+import numeral from "numeral";
+
+import styles from "./styles.module.css";
+
+interface RewardTickProps {}
```

Comment:
Can't we properly type the props here?

Reply:
Fixed in 0edb2c51d297f15fb54f2bfbba97283afc4c78e6.